### ISSUE TITLE
Project sync fails when looking for collections declared in the requirements

### ISSUE THREAD

**Issue by laurent-indermuehle** [redhat-cop/aap_configuration_template#51]
When installing the project "config_as_code", a task fetch the collections defined in `collections/requirements.yml`.
In particular, those:
```yaml
  - name: infra.controller_configuration
  - name: infra.ah_configuration
  - name: infra.ee_utilities
  - name: infra.aap_utilities
 ```

I've removed those collections from the 'community' repository because they were colliding with the same collections installed from the `validated`repository. The uniqueness constraint bug is present in AAP 2.4-6 (AH 4.9.1) and a fix is yet to be released.

In the meantime, the task is failing with this error: `ERROR! Failed to resolve the requested dependencies map. Could not satisfy the following requirements:\n* infra.controller_configuration:2.6.0 (direct request)`.

This is because I need to have access to the `validated` repository that hosts those infra collections.

So in conclusion, I think that the `validated` repository should be enabled in the controller to cover this scenario.

**Comment by laurent-indermuehle** [comment]
~~Did published content got renamed into validated at some point? Because I see contradictory informations out there: ~~

## from https://access.redhat.com/solutions/6936761
> Private Hub - **Publish**  
> ==>> Galaxy Server Url: https://PRIVATE-HUB-IP/api/galaxy/content/published/
> 
> Private Hub - **RH-certified**  
> ==>> Galaxy Server Url: https://PRIVATE-HUB-IP/api/galaxy/content/rh-certified/


## from https://console.redhat.com/ansible/automation-hub/token
> Use this URL to configure the API endpoints that clients need to download **certified** content from Automation Hub.
> https://console.redhat.com/api/automation-hub/content/published/
>
> Use this URL for **validated** content from Automation Hub.
> https://console.redhat.com/api/automation-hub/content/validated/

EDIT: Thanks to @djdanielsson I now understand that published is a private repository on our private AH and rh-certified is "published" by Red Hat on console.redhat.com.
